R/checkers.R

```diff
@@ -0,0 +1,207 @@
+#' Check if `<data.frame>` defining age-stratified hospitalisation or death risk
+#' is correct
+#'
+#' @param x A `<data.frame>`.
+#'
+#' @return A `<data.frame>`, also called for error side-effects when input is
+#' invalid.
+#' @keywords internal
+.check_risk_df <- function(x, age_range) {
```

Comment:
Suggest named vector `c(lower = x, upper = y)` as `age_range` input

Reply:
Resolved in #81.